packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -106,6 +106,33 @@ async function defaultCreateFeedItems({
       const $ = cheerioLoad(content);
 
       const link = normalizeUrl([siteUrl, permalink]);
+
+      $(`div#${blogPostContainerID} a, div#${blogPostContainerID} img`).each(
+        (_, elm) => {
+          if (elm.tagName === 'a') {
+            const {href} = elm.attribs;
+            if (href) {
+              elm.attribs.href = String(new URL(href, link));
+            }
+          } else if (elm.tagName === 'img') {
+            const {src, srcset} = elm.attribs;
+            if (src) {
+              elm.attribs.src = String(new URL(src, link));
+            }
+            if (srcset) {
+              elm.attribs.srcset = srcset
+                .split(',')
+                .map((s) => {
+                  const [imageURL, ...descriptors] = s.trim().split(/\s+/);
+                  const newImageURL = new URL(imageURL ?? '', link).href;
+                  return [newImageURL, ...descriptors].join(' ');
+                })
+                .join(', ');
```

Comment:
Looks a bit unsafe/risky, maybe introduce a dedicated lib to manipulate srcset reliably instead? see https://www.npmjs.com/package/srcset

Reply:
Thanks. I'll look into it and revise my code

Reply:
Done and since latest version of srcset is pure ESM, so I have to use the previous version.